### ISSUE TITLE
Speed up unit tests.

### ISSUE THREAD

**Issue by 36000** [yeatmanlab/pyAFQ#355]
We could reduce the number of seeds / ROIs in some key tests. Maybe mark other tests as slow. Any ideas are welcome.

**Comment by 36000** [comment]
I wonder if we could further parallelize the tests, like how python versions 3.6 and 3.7 are already done in parallel.
